spec/message.ebnf

```diff
@@ -29,12 +24,12 @@ TextChar ::= AnyChar - ('{' | '}' | Esc)
 AnyChar ::= [#x0-#x10FFFF] - [#xD800-#xDBFF]
```

Comment:
```suggestion
AnyChar ::= [#x0-#x10FFFF] - [#xD800-#xDFFF]
```

Fix the bug we just removed :-).

Reply:
Ah, good catch, thanks. Rebased on top of the current `main` and force-pushed.